dashboard/app/assets/stylesheets/application.css.scss

```diff
@@ -1366,6 +1365,12 @@ a.download-video {
   }
 }
 
+@mixin yTranslation($offset) {
```

Comment:
interesting, i didn't realise we had to create our own mixin for this.  scss doesn't already have support?